### PR TITLE
fix: add missing target attribute to menu link

### PR DESCRIPTION
**Pull request by edalex-yinzi**
##### Checklist

- [x] the [contributor license agreement][] is signed
- [x] commit message follows [commit guidelines][]
- [ ] tests are included
- [ ] screenshots are included showing significant UI changes
- [ ] documentation is changed or added

##### Description of change
Add missing `target` attribute to menu link.

[contributor license agreement]: https://www.apereo.org/node/676
[commit guidelines]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: react-front-end/tsrc/mainui/MainMenu.tsx [hunk 1]
```diff
@@ -54,6 +54,7 @@ const MainMenu = ({ menuGroups, onClickNavItem }: MainMenuProps) => {
     <ListItem
       component="a"
       href={item.href}
+      target={item.newWindow ? "_blank" : ""}
```
Comment (edalex-ian) [inline react-front-end/tsrc/mainui/MainMenu.tsx:57]: If `target` is set to empty string (`""`) does it get added to the DOM or just ignored?

Comment (PenghaiZhang) [reply react-front-end/tsrc/mainui/MainMenu.tsx:57]: Why not using `_self`?

Comment (edalex-yinzi) [reply react-front-end/tsrc/mainui/MainMenu.tsx:57]: Looks like the browser automatically set "_self" value to target.

Comment (edalex-yinzi) [reply react-front-end/tsrc/mainui/MainMenu.tsx:57]: Showing this: 
image

**Review by edalex-ian — APPROVED** [review]
I'm happy with this. I was thinking it'd be good to have it as undefined so we don't unnecessarily add attributes to the DOM, but hey, this'll do. :wink:

**Review by PenghaiZhang — APPROVED** [review]

**Pull request merged**